services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str | None = None
+) -> None:
```

Comment:
Invalid type hint for return type
```suggestion
) -> Order:
```